README.rst

```diff
@@ -397,6 +397,10 @@ some system-wide directory like ``/var/zopebackups/instancename/`` and
     Location where backups are stored. Defaults to ``var/backups`` inside the
     buildout directory.
 
+``locationprefix``
```

Comment:
probably @mauritsvanrees has more a saying here, but shouldn't it go with an underscore like all other options (I see a bit below ``only_blobs`` for example